memory/dma_memtest.py

```diff
@@ -41,6 +41,8 @@ class DmaMemtest(Test):
 
     :avocado: tags=memory,privileged
     """
+    global cancel_flag 
```

Comment:
use this as self variable as scope will be visible inside class

Reply:
ok will do that

Reply:
Instead of such a complex check can't we just check for tmp_dir in teardown()